### PR TITLE
Opsec audit script

### PR DESCRIPTION
**Pull request by timmoreton**
### Description

A script for testing aspects of a deployment's security. 
  
### Tested

On centos, debian.

### Backwards compatibility

Yes

### CODE REVIEW

File: packages/opsec-audit/opsec-audit.sh [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+echo
+sleep 3
+echo "###############################################"
+echo "You Are $HOSTNAME..:"
+echo
+sleep 3
+echo "Starting"
+START=$(date +%s)
+echo -e "\e[0;33m 1. Downloading Lynis////// \e[0m"
+curl https://downloads.cisofy.com/lynis/lynis-2.7.5.tar.gz -o lynis-2.7.5.tar.gz
```
Comment (tkporter) [inline packages/opsec-audit/opsec-audit.sh:25]: What if this a proxied validator that's entirely disconnected from the outside world?

Comment (timmoreton) [reply packages/opsec-audit/opsec-audit.sh:25]: Great question. Think we are going to change this so they download the script ahead of time and we just check its md5

Comment (jcortejoso) [reply packages/opsec-audit/opsec-audit.sh]: Maybe packing the script as a container would be a solution, so we do not need to download anything

**Comment by barbaraliau** [comment]
Is this still relevant or should this be closed?

**Pull request closed without merging**